Cargo.toml

```diff
@@ -19,6 +19,7 @@ ring = "0.17"
 rustls = "0.23"
```

Comment:
note: small typo in your commit message for 83d9b88efa8d8ef3bfb18211765262ac3f5c718e:

"On Unix, certifictes were " -> "certifictes" replaced with "certificates"

Reply:
Done